kubeshop/botkube-docs
# Examples and tutorials

## Description

Changes proposed in this pull request:

- adding examples and tutorials section to docs 
- 

## Testing


- run npm start 
- chose the unreleased version
- check the content and formatting

## Related issue(s)



https://github.com/kubeshop/botkube-cloud/issues/880
